config.py

```diff
@@ -122,7 +119,7 @@ class Test(Config):
     SHARED_EMAIL_KEY = "KEY"
 
     DM_MAILCHIMP_USERNAME = 'not_a_real_username'
-    DM_MAILCHIMP_API_KEY = 'not_a_real_key'  # pragma: allowlist secret
+    DM_MAILCHIMP_API_KEY = 'not_a_real_key'
```

Comment:
We might need to keep this pragma?

Reply:
Ah yes. Am I missing something that generates / checks for these?

Reply:
Have you got https://github.com/alphagov/gds-pre-commit set up on your machine?

Reply:
No! That'd do it